lib/aws/ec2.js

```diff
@@ -48,30 +55,60 @@ function AWSEC2() {
             return;
         }
 
-        self.callbackFind = function(data) {
-            var hasRules = false;
-            if (data.SecurityGroups.length !== 0) {
-                for( var item in data.SecurityGroups) {
-                    var permissions = data.SecurityGroups[item].IpPermissions;
-                    console.log("Inbound Rules");
-                    for ( var it in permissions) {
-                        hasRules = true;
-                        console.log(data.SecurityGroups[item].IpPermissions[it]);
-                    }
-                    var permissionsEgress = data.SecurityGroups[item].IpPermissionsEgress;
-                    console.log("Outbound Rules");
-                    for ( var it2 in permissionsEgress) {
-                        hasRules = true;
-                        console.log(data.SecurityGroups[item].IpPermissionsEgress[it2]);
-                    }
+        self.getEvaluations = function(data){
+            var annotation = "",
+                complianceType = EVALUATION_TYPES.COMPLAINT,
+                evaluations = [];
+
+            for (var idx in data){
+                var ingressPerm = rules[idx].IpPermissions,
+                    egressPerm = rules[idx].IpPermissionsEgress,
+                    vpcId = rules[idx].VpcId,
+                    sgId = rules[idx].GroupId;
+
+                console.log("=======Inbound Rules for VPC ["+ vpcId +"]=======");
+                if (ingressPerm.length !==0){
+                    annotation += "["+ ingressPerm.length +"] ingress rules present for vpc ["+ vpcId +"] \n";
+                    complianceType = EVALUATION_TYPES.NON_COMPLIANT;
```

Comment:
It is if there are rules in default security group the config is notified that the group is non compliant